doc/whats_new.rst

```diff
@@ -20,6 +20,7 @@ BUG
 API
 ~~~
 
+    - Added logscale epoch plotting option via `logscale=True` in :func:`mne.viz.plot_epochs_image`, by `Sagun Pai`_
```

Comment:
Sphinx markdown will not display the url properly if you don't add your homepage at the bottom of the page. Also, I think the latest changes are at the bottom, not the top of the list.